### PR TITLE
Addition of tyaml's README + licensing disclaimer

### PR DESCRIPTION
**Pull request by gbgabo**
PR focused on creating a first README of tyaml + disclaimer of licensing on the scripts source code header

### CODE REVIEW

File: tyaml/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# `tyaml`
+
+A simple script to get values on a yaml file.
```
Comment (calebe94) [inline tyaml/README.md]: "A simple script to parse a yaml file." Would be better.

**Review by calebe94 — CHANGES_REQUESTED** [review]
Just a change in the description

File: tyaml/README.md (hunk 1 above)
Comment (gbgabo) [reply tyaml/README.md]: Indeed!
Just did it!

**Review by calebe94 — APPROVED** [review]
Everithing seems OK!

**Pull request merged**